### PR TITLE
updates

### PR DESCRIPTION
**Pull request by viliusle**
#63 - better demo
#72 fixed building on Windows, also "npm run dev-modern"
#68 - Documentation improvements (data attributes) - already on your production branch
iframe improvements

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -1,23 +1,186 @@
...
+		</tr>
+	</table>
+
 
 	<script src="../dist/tobii.min.js"></script>
 	<script>
-		const tobii = new Tobii()
+
+		//prepare manual executions
+        var manual = document.querySelector('#manual');
```
Comment (midzer) [inline demo/index.html]: Prefer `const` and `document.getElementById()`

File: demo/index.html [hunk 2]
```diff
@@ -1,23 +1,186 @@
...
 	<meta charset="utf-8">
-	<title></title>
-	<meta name="description" content="">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Tobii demo</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
 	<link rel="stylesheet" href="../dist/tobii.min.css">
+	<style>
+		body{
+            max-width: 1000px;
```
Comment (midzer) [inline demo/index.html]: Please check the intendation (4 spaces)

File: demo/index.html [hunk 3]
```diff
@@ -1,23 +1,186 @@
...
 <head>
 	<meta charset="utf-8">
-	<title></title>
-	<meta name="description" content="">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Tobii demo</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
 	<link rel="stylesheet" href="../dist/tobii.min.css">
+	<style>
+		body{
```
Comment (midzer) [inline demo/index.html]: Personally, I prefer a space behind the selector...

File: src/js/index.js [hunk 4]
```diff
@@ -326,6 +330,8 @@ export default function Tobii (userOptions) {
 
       onCleanup (container) {
         // Nothing
+        var video = container.getElementsByTagName('iframe')[0]
```
Comment (midzer) [inline src/js/index.js]: `const` here as well

File: package.json [hunk 5]
```diff
@@ -38,10 +39,11 @@
   },
   "scripts": {
     "build": "npm run clean && npm run build:main && npm run build:browser",
-    "build:main": "BROWSERSLIST_ENV=main microbundle build --raw --no-sourcemap --name Tobii",
-    "build:browser": "BROWSERSLIST_ENV=browser microbundle build --raw -f iife src/js/browser.js -o dist/tobii.min.js --no-sourcemap --name Tobii",
+    "build:main": "cross-env BROWSERSLIST_ENV=main microbundle build --raw --no-sourcemap --name Tobii",
```
Comment (midzer) [inline package.json:42]: Thanks for fixing windows build!

File: package.json [hunk 6]
```diff
@@ -38,10 +39,11 @@
...
+    "build:browser": "cross-env BROWSERSLIST_ENV=browser microbundle build --raw -f iife src/js/browser.js -o dist/tobii.min.js --no-sourcemap --name Tobii",
     "clean": "rimraf dist",
     "dev": "microbundle watch --raw --format cjs",
+    "dev-modern": "microbundle watch --raw --format modern",
```
Comment (midzer) [inline package.json:46]: Great addition!

File: demo/modern.html [hunk 7]
```diff
@@ -1,23 +1,186 @@
...
+		</tr>
+	</table>
+
 
 	<script type="module">
-		import Tobii from '../dist/tobii.modern.js';
+      	import Tobii from '../dist/tobii.modern.js';
+
+		//prepare manual executions
+        var manual = document.querySelector('#manual');
```
Comment (midzer) [inline demo/modern.html]: `const` and `document.getElementById()` again

File: demo/modern.html [hunk 8]
```diff
@@ -1,23 +1,186 @@
...
 <head>
 	<meta charset="utf-8">
-	<title></title>
-	<meta name="description" content="">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
+	<title>Tobii demo (modern)</title>
+	<meta name="viewport" content="width=device-width, initial-scale=1" />
 	<link rel="stylesheet" href="../dist/tobii.min.css">
+	<style>
+		body{
```
Comment (midzer) [inline demo/modern.html]: Intendation and spacing comments from `demo.html` apply here too

File: demo/index.html (hunk 2 above)
Comment (midzer) [reply demo/index.html]: (in whole file)

**Review by midzer — CHANGES_REQUESTED** [review]
First of all, big ups for your contributions! This is really massive.

Some minor style changes, sorry. Otherwise it looks really good!

This will land in a new release soon.

**Comment by viliusle** [comment]
OK, but some lint rules are wrong. For example saying, that commands `font-weight: bold;` or `background: #ffffff;` or `color: #55aa22;` are incorrect, is silly... This makes code writing much slower.

Also JS without ";" and tab size with only 2 spaces is very strange for me, but I will try to adapt to these. Basically code auto-format on save works well.

Can I submit issue about some lint rules?

**Review by midzer — APPROVED** [review]
Looks good to me!

**Comment by midzer** [comment]
If you want to improve lint rules, feel free to open an issue, @viliusle

**Pull request merged**